### DISCUSSION TITLE
React-native-chat-ui: Image Picker Not Working with Expo SDK

### DISCUSSION THREAD

**Discussion by mirsahib** [flyerhq/react-native-chat-ui#93] [Q&A] [answered]
Hi 
I'm using the react-native-chat-ui library in a project with the Expo SDK. The chat handles text messages perfectly, but I'm having trouble with the image picker—it won't open using the code from the documentation. Does anyone know if react-native-chat-ui fully supports Expo SDK? If so, are there any specific configurations needed to get the image picker working? Thanks!

**Comment by demchenkoalex** [comment, marked as answer]
Hey :) as of right now this project is not maintained and I advise not to use it. You can browse through the code if you like to see the implementation details, maybe it will give you an idea, however if something will require any work - it won't be done.
